### ISSUE TITLE
“Select your county” button has an insufficient accessible name and lacks a role of button [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#197]
The “select your county” button found on the “NY COVID Data” screen is labeled by whichever county is currently selected (e.g., “Albany”). Without the visual information provided by the “filter” icon and the surrounding content, VoiceOver users are unlikely to understand the purpose of the button without activating it and exploring the next screen. The button also does not convey a role of button.

![Button used to trigger overlay for selecting which counties’ data to display](https://user-images.githubusercontent.com/4437084/91873678-4d2f4c80-ec47-11ea-9d79-9335794d50d6.png)

The accessibilityLabel attribute can be used to provide additional information to VoiceOver users. As the accessibilityLabel overwrites the visible text label for said users, the accessibilityLabel value must include the visible text. Use the accessibilityRole attribute on the interactive element (TouchableOpacity, Pressable, etc.) to convey a button role to VoiceOver users.

```javascript
<TouchableOpacity
  onPress={…}
  accessibilityRole='button'
  accessibilityLabel='[Insert additional text here for VoiceOver users] All Counties in New York'
>
  <Text>All Counties in New York</Text>
</TouchableOpacity>
```

**Comment by fahd-arshad** [comment]
As per @willslone this needs to be addressed before launch. Upgrading severity to Critical accordingly.

**Comment by floridemai** [comment]
@willslone Should be addressed in build 34 and after.
